client/webserver/webserver.go

```diff
@@ -162,6 +165,30 @@ type clientCore interface {
 	SendShielded(appPW []byte, assetID uint32, toAddr string, amt uint64) ([]byte, error)
 }
 
+// genCertPair generates a key/cert pair to the paths provided.
+func genCertPair(certFile, keyFile string, altDNSNames []string) error {
+	log.Infof("Generating TLS certificates...")
+
+	org := "dex webserver autogenerated cert"
+	validUntil := time.Now().Add(2 * 365 * 24 * time.Hour)
```

Comment:
Browsers seem to dislike validity dates farther than 2 years and a few months out.